app-misc/khard/khard-0.18.0.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
```

Comment:
You may just use EAPI=8 as per recommendations of [DevManual](https://devmanual.gentoo.org/ebuild-writing/eapi/index.html)

Reply:
Habits die hard, I changed it locally. Actually I prefer to see that EAPIs are actually strings, but as the rest of the Gentoo world abuses them as numbers anyways there is no real benefit.